linchpin/shell/__init__.py

```diff
@@ -208,8 +208,11 @@ def init(ctx):
               cls=MutuallyExclusiveOption, mutually_exclusive=["run_id"])
 @click.option('-if', '--inventory-format', default="cfg",
               help="Inventory format can be cfg or json")
+@click.option('-rhof', '--run-hooks-on-failure', is_flag=True, default=None)
+@click.option('-nh', '--no-hooks', is_flag=True, default=None)
```

Comment:
Any long switches must have two dashes (eg. `--nh`)

Reply:
@samvarankashyap I updated your code with the `--nh` and `--ifh` and they work just fine. I'm not sure why they don't work for you. Please change this back.